### PR TITLE
Cleanup existing gateway if launching the boat: fixes issue#11.

### PR DESCRIPTION
**Pull request by holdenk**
Allow boat launching after running `sc.stop()`.

### CODE REVIEW

File: coffee_boat/captain.py [hunk 1]
```diff
@@ -75,6 +75,17 @@ def launch_ship(self):
     .. note::
        This function must be called before your init your SparkContext!"""
     self._raise_if_running()
+
+    # Doing sketchy things with the gateway
```
Comment (rgbkrk) [inline coffee_boat/captain.py:79]: What's the implication of this?

Comment (holdenk) [reply coffee_boat/captain.py:79]: So we need to shut down the existing JVM process since that's (currently) the only way the PYTHON_PATH can be updated.

Comment (rgbkrk) [reply coffee_boat/captain.py:79]: No way to change environment variables for the JVM process? That's too bad, though entirely sensible.

Comment (holdenk) [reply coffee_boat/captain.py:79]: So the problem is it's cached inside a val inside of a class somewhere. We could maybe do something to change it, but I think that would be even sketchier than just bouncing the JVM.

**Pull request merged**